src/gwt/src/org/rstudio/studio/client/workbench/views/jobs/view/JobProgress.java

```diff
@@ -89,6 +106,15 @@ else if (job.max > 0)
             status = job.status;
          }
       }
+
+      // show stop button if job has a "stop" action, and is not completed
+      if (job.completed == 0)
+         stop_.setVisible(
+               JsArrayUtil.jsArrayStringContains(job.actions, JobConstants.ACTION_STOP) &&
+               job.completed == 0);
```

Comment:
No need for this test since it's inside an if statement that already guarantees the condition.

Reply:
Fixed 👍